src/Core/SettingsChangesHistory.cpp

```diff
@@ -49,6 +49,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"correlated_subqueries_substitute_equivalent_expressions", false, true, "New setting to correlated subquery planning optimization."},
             {"function_date_trunc_return_type_behavior", 0, 0, "Add new setting to preserve old behaviour of dateTrunc function"},
             {"output_format_parquet_geometadata", false, true, "A new setting to allow to write information about geo columns in parquet metadata and encode columns in WKB format."},
+            {"vector_search_with_rescoring", true, false, "New setting."},
```

Comment:
In this PR, rescoring is on, i.e. we did not change the previous behavior. Settings changes history should therefore record this:

```cpp
{"vector_search_with_rescoring", true, true, "New setting."},
```